foca/database/register_mongodb.py

```diff
@@ -35,9 +34,7 @@ def register_mongodb(
             # Instantiate PyMongo client
             mongo = create_mongo_client(
                 app=app,
-                host=conf.host,
-                port=conf.port,
-                db=db_name,
+                settings=conf
```

Comment:
Really not a big fan of this. I intentionally left the API simple so that it can be easily used separately from the rather complex FOCA config setup. Couldn't you just pass what you need from the config?